src/bitmap/inherent.rs

```diff
@@ -375,8 +386,6 @@ impl RoaringBitmap {
     /// assert_eq!(rb.rank(10), 2)
     /// ```
     pub fn rank(&self, value: u32) -> u64 {
-        // if len becomes cached for RoaringBitmap: return len if len > value
```

Comment:
This is incorrect. Should have been return len if len > max

Computing max is not constant time for `BitmapStore`